src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -87,6 +87,18 @@ const ContactDetails = ({
       />
     ));
 
+  const saveExitLink = (() => {
+    let link = '';
+    if (systemIntake.requestType === 'SHUTDOWN') {
+      link = '/';
+    } else {
+      link = flags.taskListLite
+        ? `/governance-task-list/${systemIntake.id}`
+        : '/';
+    }
+    return link;
+  })();
```

Comment:
I chose to go with an immediately invoked function expression because it doesn't require an argument and it reads nicer.

The `history.push(saveExitLink)` receives a `string` rather than a function that returns a `string`.

Any objections?

Reply:
No objections!